dapp/src/utils/network.js

```diff
@@ -11,7 +11,7 @@ const blockExplorers = {
 
 export const getBlockExplorerUrl = (networkType) => blockExplorers[networkType]
 
-export const getApiRoot = (networkType) => CONFIG.api.url[networkType]
+export const getApiRoot = (networkType) => CONFIG.api.url[networkType === 'fuse' ? 'default' : networkType]
```

Comment:
I think if you are on Fuse, this doesn't mean you need to go to the default server.
If the community issued on mainnet (and default is ropsten) you do need to go to mainnet server